src/enrollments/enrollments.controller.ts

```diff
@@ -18,4 +19,13 @@ export class EnrollmentsController {
   enrollStudent(@Body() createEnrollmentDto: CreateEnrollmentDto): Promise<EnrollmentResponseDto> {
     return this.enrollmentsService.enrollStudent(createEnrollmentDto);
   }
+
+  @Roles(EUserType.TEACHER)
+  @Delete()
+  unenrollStudent(
+    @CurrentUser() user: { id: number },
+    @Body() deleteEnrollDto: CreateEnrollmentDto,
+  ): Promise<void> {
+    return this.enrollmentsService.removeStudent(user.id, deleteEnrollDto);
+  }
```

Comment:
I think this should be inside the classrooms controller

Reply:
Bhaiya, all the enrollments and unenrollments of students are kept in this controller. I have done this for ease of use.

Reply:
Moving the endpoints to classrooms controller wouldn't be much of a difference. You'd just have to call the same service method in that file.
And please follow REST principles and use endpoints such as: `classrooms/:classroomId/enroll`, 
`classrooms/:classroomId/students` etc.

Reply:
Okay bhaiya. I will create a separate ticket for moving all the enrollment controller to classroom controller.